Web_Development/Javascript/Topics/5.Syntax/Objects/README.md

```diff
@@ -0,0 +1,136 @@
+# JavaScript Objects
+An JavaScript object is anything(entity) having properties and methods.
+
+JavaScript is an object-based language, almost everything is an object. For eg:- numbers, strings etc are objects if they are defined with the ```new``` keyword.
```

Comment:
object oriented will be more effective than object based

Reply:
@AbhinandanAdhikari Yes, well done your PRs look good from my side. Wait till others mentor approve it:)

Reply:
@s4-2 @Creatoon @dakshsethi Mentors! Made the changes, please review it.